content/latest/quick-start/docker/explore-ysql.md

```diff
@@ -23,7 +23,7 @@ docker cp ./data/reviews.sql yb-tserver-n1:/home/yugabyte/data/.
 docker cp ./data/users.sql yb-tserver-n1:/home/yugabyte/data/.
 ```
 
-Run psql to connect to the service.
+Run ysqlsh to connect to the service.
 
 ```sh
 $ docker exec -it yb-tserver-n1 /home/yugabyte/bin/ysqlsh -h yb-tserver-n1 -p 5433 -U postgres  --echo-queries
```

Comment:
Are any of these defaulted in ysqlsh now? I feel the default port should be 5433 and the user postgres for ysqlsh. This might already be done, in which case this command is much simpler. Same for the other sections.

Reply:
Yes, -p and -U default to 5433 and postgres respectively. We don’t need to specify those here.